src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -152,12 +152,6 @@ void ManageZoneEquipment(EnergyPlusData &state,
     // PURPOSE OF THIS SUBROUTINE:
     // Calls the zone thermal control simulations and the interfaces (water-air, refrigerant-air, steam-air, electric-electric, water-water, etc)
 
-    if (state.dataZoneEquipmentManager->GetZoneEquipmentInputFlag) {
-        GetZoneEquipment(state);
-        state.dataZoneEquipmentManager->GetZoneEquipmentInputFlag = false;
-        state.dataZoneEquip->ZoneEquipInputsFilled = true;
-    }
-
```

Comment:
Move from ManageZoneEquipment to HeatBalanceManager::GetBuildingData. That is certainly earlier in the simulation. And then you removed all the other calls to GetZoneEquipmentData. You Sir, have nerves of steel!

Reply:
That's what I was thinking!